packages/buckets-node-sync/src/cli.ts

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env node
+
+import yargs, {Argv, string} from "yargs"
+import { execute, RunOutput } from './index'
+
+// Example:
+// buckjs push --api=http://127.0.0.1:3007 -k blp5ihqcs5zdryhc23fx2x7l5ru -s b6dtmohcxvcd3uqnbpx6b7ygb6m7hu4cslgsoroy -t bafku6ozemzot6fwazeolxag2kvd3c753nudnwat4mfoinatiffeicdi -n test -p /Users/andrewhill/Textile/textile/js-textile/packages/buckets-node-sync/test/website
```

Comment:
So cool!

Reply:
just pushed an update that cleans it all up. allows same ENV var pattern etc (see readme)

Reply:
<img width="905" alt="Screen Shot 2021-01-22 at 10 15 49 AM" src="https://user-images.githubusercontent.com/370259/105529155-d7ca2400-5c9a-11eb-85f2-71dd8cadfe20.png">

Reply:
those aren't really keys/secrets, don't worry :D